pyramid-app/{{ cookiecutter.slug }}/{{ cookiecutter.package_name }}/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+def includeme(_config):  # pragma: no cover
+    pass
```

Comment:
In freshly-created Pyramid apps there needs to be a `models/__init__.py` with an empty `includeme()` otherwise things crash. But `make template` won't create/update this file in existing projects